app/controllers/browse_controller.rb

```diff
@@ -1,60 +1,19 @@
 class BrowseController < ApplicationController
-  enable_request_formats show: [:json]
-
   def index
     page = MainstreamBrowsePage.find("/browse")
     @dimension26 = 1
     @dimension27 = page.top_level_browse_pages.count || 0
+    template = :index
     setup_content_item_and_navigation_helpers(page)
-    index_html(page)
+    slimmer_template "gem_layout_full_width"
```

Comment:
As both the `index` and `show` methods use the same slimmer template, this can be set once straight after the class is set up - for example:

```ruby
class BrowseController < ApplicationController
  slimmer_template "gem_layout_full_width"

  def index
    ...
```